pkg/app/pipectl/cmd/event/register.go

```diff
@@ -43,7 +44,7 @@ func newRegisterCommand(root *command) *cobra.Command {
 
 	cmd.Flags().StringVar(&r.name, "name", r.name, "The name of event.")
 	cmd.Flags().StringVar(&r.data, "data", r.data, "The string value of event data.")
-	// TODO: Allow specifying event labels.
+	cmd.Flags().StringToStringVar(&r.labels, "labels", r.labels, "The list of labels for event. Format: key=value,key2=value2 .")
```

Comment:
nit: In that case, I feel like the period at the end of a sentence is unneeded.

Reply:
😄

Reply:
ww. Sure.